extensions/amp-cache-url/0.1/amp-cache-url.js

```diff
@@ -14,11 +14,27 @@
  * limitations under the License.
  */
 
+import * as ampToolboxCacheUrl from '@ampproject/toolbox-cache-url';
+import {urls} from '../../../src/config';
+
 export class AmpCacheUrlService {
   /**
    * Create cache url service
    */
   constructor() {}
+
+  /**
+   *
+   * @param {string} url
+   * @param {string=} cache the suffix to the cache URL (eg: cdn.approject.org)
+   * @return {!Promise<string>}
+   */
+  createCacheUrl(url, cache = urls.cdn) {
+    return ampToolboxCacheUrl.createCacheUrl(
+      cache.replace(/https?:\/\//, ''),
+      url
```

Comment:
Would this be accurate?

```suggestion
   * @param {string=} cacheDomain the cache domain name (eg: cdn.approject.org)
   * @return {!Promise<string>}
   */
  createCacheUrl(url, cacheDomain = urls.cdn) {
    return ampToolboxCacheUrl.createCacheUrl(
      cacheDomain.replace(/https?:\/\//, ''),
      url
```